decidim-admin/app/views/decidim/admin/participatory_process_steps/edit.html.erb

```diff
@@ -0,0 +1,11 @@
+<% content_for :title do %>
+  <h2><%= t ".title" %></h2>
+<% end %>
+
+<%= form_for(@form, url: participatory_process_step_path(@participatory_process_step.participatory_process, @participatory_process_step)) do |f| %>
```

Comment:
Can't we use `url_for` or just `@participatory_process_step` to make it shorter?